Tools/install.sh

```diff
@@ -22,6 +22,46 @@ FONT_PACKAGES=(\
     fonts-droid-fallback \
     )
 
+check_for_installed_tools () {
+    REQUIRED_TOOLS="${@}"
```

Comment:
shellcheck: `SC2124 (warning): Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.`